frontend/src/components/organisms/About.tsx

```diff
@@ -1,108 +1,177 @@
+            Yes, publish
+          </Button>
         </Grid>
       </Grid>
     </div>
   );
 };
 
+type aboutPageData = {
+  pageid: string;
+  content: string
+};
+
 /**
  * An About component
  */
 const About = ({ edit }: AboutProps) => {
-  var default_text = `
-  <h2>About Lagos Food Bank</h2>
-  <h3>Mission</h3>
-  <p>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-    eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
-    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-    aliquip ex ea commodo consequat. Duis aute irure dolor in
-    reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
-    pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
-    culpa qui officia deserunt mollit anim id est laborum.
-  </p>
-  <h3>Why Volunteer?</h3>
-  <p>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-    eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
-    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-    aliquip ex ea commodo consequat. Duis aute irure dolor in
-    reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
-    pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
-    culpa qui officia deserunt mollit anim id est laborum.
-  </p>
-  <h2>Sign Up Process</h2>
-  <p>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-    eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
-    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-    aliquip ex ea commodo consequat. Duis aute irure dolor in
-    reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
-    pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
-    culpa qui officia deserunt mollit anim id est laborum.
-  </p>
-  <h2>Programs</h2>
-  <h2>Certificate Request Process</h2>
-  <p>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-    eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
-    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-    aliquip ex ea commodo consequat. Duis aute irure dolor in
-    reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
-    pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
-    culpa qui officia deserunt mollit anim id est laborum.
-  </p>
-  <h2>Request Certificate</h2>
-  <p>Fill out this form. Login first</p>
-  `;
-  const [value, setValue] = useState(`${default_text}`);
-  const [open, setOpen] = React.useState(false);
+
+  /** Tanstack query for fetching the about page data */
+  const {
+    data: aboutPageDetailsQuery,
+    isPending: aboutPageFetchPending,
+    isError,
+  } = useQuery({
+    queryKey: ["about"],
+    queryFn: async () => {
+      const { response, data } = await api.get("/about");
+      // console.log(data);
+      setValue(data.content);
```

Comment:
the beauty of tanstack query is that you don't need to manage states like this, so you actually don't need this. The `data` holds all the information you need,

Reply:
hi Akin; I believe we use the useState hook to manage the data in the frontend so that we can store the edits that the user makes in ReactQuill and then call the updateAboutPage on that data so that the content is synced appropriately between the front- and backends. I'm not sure how else I'd be able to keep track of the user's edits without using useState; unless I'm missing something?

`        <ReactQuill
          theme="snow"
          value={value}
          onChange={setValue}
          readOnly={false}
        />`